Config/secrets.xcconfig.example

```diff
@@ -1 +1,3 @@
 POCKET_API_CONSUMER_KEY=test-consumer-key
+BRAZE_API_KEY=123123123123-zcsd-132123-csd
```

Comment:
Just double checking that this is not the real key?

Reply:
no its not. its an example